kep/594-resourcepolicy/README.md

```diff
@@ -0,0 +1,191 @@
+# Resource Policy
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Use Cases](#use-cases)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [CRD API](#crd-api)
+  - [Implementation Details](#implementation-details)
+    - [Scheduler Plugins](#scheduler-plugins)
+      - [PreFilter](#prefilter)
+      - [Filter](#filter)
+      - [Score](#score)
+    - [Resource Policy Controller](#resource-policy-controller)
+- [Known limitations](#known-limitations)
+- [Test plans](#test-plans)
+- [Graduation criteria](#graduation-criteria)
+- [Feature enablement and rollback](#feature-enablement-and-rollback)
+<!-- /toc -->
+
+## Summary
+This proposal introduces a plugin to allow users to specify the priority of different resources and max resource 
+consumption for workload on differnet resources.
+
+## Motivation
+The machines in a Kubernetes cluster are typically heterogeneous, with varying CPU, memory, GPU, and pricing. To 
+efficiently utilize the different resources available in the cluster, users can set priorities for machines of different 
+types and configure resource allocations for different workloads. Additionally, they may choose to delete pods running 
+on low priority nodes instead of high priority ones. 
+
+### Use Cases
+
+1. As a user of cloud services, there are some stable but expensive ECS instances and some unstable but cheaper Spot 
+instances in my cluster. I hope that my workload can be deployed first on stable ECS instances, and during business peak 
+periods, the Pods that are scaled out are deployed on Spot instances. At the end of the business peak, the Pods on Spot 
+instances are prioritized to be scaled in.
+
+### Goals
+
+1. Develop a filter plugin to restrict the resource consumption on each unit for different workloads.
+2. Develop a score plugin to favor nodes matched by a high priority unit.
+3. Automatically setting deletion costs on Pods to control the scaling in sequence of workloads through a controller.
+
+### Non-Goals
+
+1. Modify the workload controller to support deletion costs. If the workload don't support deletion costs, scaling in 
+sequence will be random.
+2. When creating a ResourcePolicy, if the number of Pods has already violated the quantity constraint of the 
```

Comment:
could you reword this to be more readable? basically, just use a positive tone to describe it as a goal that is out of scope of this KEP.